builds/e2e/stresstest.yaml

```diff
@@ -0,0 +1,155 @@
+trigger: none
+pr: none
+
+variables:
+  azure.keyVault: 'edgebuildkv'
+  azure.subscription: 'azureiotedge-arm'
+  Build.SyncSources: false
+  edgelet.package.build: 55463
+  images.artifact.name: 'core-linux'
+  images.build: 55174
+  pool.name: 'Azure-IoT-Edge-Core'
+  vsts.project: $(System.TeamProjectId)
```

Comment:
We need to scrub these variables for anything that is internal: `edgebuildkv`, `azureiotedge-arm`, etc.